### PR TITLE
Deprecate PBS clients

### PR DESCRIPTION
**Pull request by cbush**
This removes bluehawk states in the RN, SwiftUI and Xamarin clients as we will not support the PBS versions anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
 # realm-template-apps
 
+This is the main repo for all MongoDB App Services & Realm template starter app
+clients and backend configurations.
+
 A GitHub action uploads to the realm-template-apps s3 bucket. See [.github/workflows/zip-everything-and-upload-to-s3.yml](.github/workflows/zip-everything-and-upload-to-s3.yml).
```
Comment (mongodben) [inline README.md:6]:
```suggestion
A GitHub action uploads to the realm-template-apps S3 bucket. See [.github/workflows/zip-everything-and-upload-to-s3.yml](.github/workflows/zip-everything-and-upload-to-s3.yml).

```

while we're in here changing things.

**Review by dacharyc — APPROVED** [review]
SwiftUI client works as expected 👍

**Comment by cbush** [comment]
RN and Xamarin checked and working

**Comment by MongoCaleb** [comment]
Xamarin (iOS and Android) LGTM

**Review by mongodben — APPROVED** [review]
lgtm!

**Comment by cbush** [comment]
Double checked manifest changes

**Comment by takameyer** [comment]
@cbush  The `generated` folder contains partition based sync templates and the flexible sync ones were deleted.

**Comment by cbush** [comment]
Correct - the flex ones now exist in the regular source (e.g. `react-native/`). The ones left in `generated/` (other than the web ones) are deprecated.

**Pull request merged**